config/src/main/resources/hoodi.json

```diff
@@ -0,0 +1,1105 @@
+{
+  "config": {
+    "chainId": 560048,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "preMergeForkBlock": 0,
+    "terminalTotalDifficulty": 0,
+    "ethash": {},
+    "shanghaiTime": 0,
+    "cancunTime": 0,
+    "pragueTime": 1742999832,
+    "blobSchedule": {
+      "cancun": {
+        "target": 3,
+        "max": 6,
+        "baseFeeUpdateFraction": 3338477
+      },
+      "prague": {
+        "target": 6,
+        "max": 9,
+        "baseFeeUpdateFraction": 5007716
+      }
+    },
+    "depositContractAddress": "0x00000000219ab540356cBB839Cbe05303d7705Fa",
```

Comment:
Just double checking the casing isn't going to matter in this context; the mainnet config uses all lower.

Might become significant for https://github.com/hyperledger/besu/pull/8417; or maybe runtime version of Address will be deserialized consistently despite this config.

Reply:
I've changed it now to match the Sepolia config. But it's true that the addresses are a mixed bag wrt to being lower or upper case